### ISSUE TITLE
Add setupApplicationTest helper file to blueprints

### ISSUE THREAD

**Issue by iezer** [emberjs/ember.js#16976]
Not sure if this needs to be an RFC but I'd like to start the issue here. I'd like to propose changing the Ember app blueprints to create a new file:

```js
// tests/helpers/setup-application-test.js
import { setupApplicationTest } from 'ember-qunit';

export default function(hooks) {
  setupApplicationTest(hook);
}
```

And change the Acceptance Test blueprint:

```js
// BEFORE
import { setupApplicationTest } from 'ember-qunit';

// AFTER
import setupApplicationTest from '[APP_NAME]/tests/helpers/setup-application-test';
```

It's common to have to do this in Ember apps when there are additional steps that have to be performed for all acceptance tests.

I also propose doing something similar for `setupRenderingTest`.

Thoughts?

**Comment by rwjblue** [comment]
Yep, totally in favor. Checkout https://github.com/ember-cli/ember-cli/pull/7657 which is roughly the same proposal.  I think having multiple exports from a single `tests/helper/index.js` leads to a nicer "feel" (and easier use in the actual blueprints), but thats a pretty small tweak to your proposal anyways.

I haven't had a chance to work on an RFC for this, but would absolutely love help if someone else does have the time...
